### DISCUSSION TITLE
Help appreciated with setup

### DISCUSSION THREAD

**Discussion by mzurhorst** [jomjol/AI-on-the-edge-device#2246] [Q&A]
Hi.

I have flashed the v15.1.1 manually and copied the files on the SD card.  
The web interface is accessible through WLAN, and I was able to take the initial setup tour to define my reference image, ROIs etc.
So, all looks pretty straight forward and promising.

However, after the reboot, nothing happens anymore.
- the overview page is blank
- manually pressing the recognitiion button tells me: "please wait for first round to complete! -- Current state: Take image"

From the documentation, I understand that the sections of the config.ini follow the order of the process steps.
When looking at the config.ini, I now realize that the [TakeImage] section is complete empty / commented-out.  When going to the settings, I cannot configure anything (see below). I was assuming this would be expert settings and a default is applied, but with the commented-out section, I am not sure anymore)

What am I doing wrong here?

Thanks and best regards,
  Marcus

![grafik](https://user-images.githubusercontent.com/575190/227765304-be8ffc00-20f7-4d47-80d7-9bab7edcc22f.png)

![grafik](https://user-images.githubusercontent.com/575190/227765222-b88b8971-65d0-4fd2-905c-063d058e8c22.png)

**Comment by mzurhorst** [comment]
There are actually some further observations:
- all the fields on the right side of the Overiew page are emty
- System/Info:  all fields are empty except "last restart", but this is showing 19700101 value
- System/OTA Update:  here, it says "Status: idle"

I have found 2 log files:
- log_1970-01-01.txt:     This has plently of success messages about WIFI, SDCARD R/W Check, Detection of PSRAM, Camera etc.    This file concludes with "Initialization completed successfully! - Starting flow task ..."
- log_2023-03-26.txt:   only errors:    File sending failed! - File does not exist /img_tmp/ref0.jpg  

The later one is interesting because those files are actually there .  I can navigate with the "File Server" to those files, click on the jpgs and display them.

**Reply by caco3** [reply]
please provide a full debug log from startup.

If the config page shows all empty stuff it also could be an Ethernet problem, reloading it should fix it.
But I have no idea how `[TakeImage]` could get disabled. By default it is enabled, see https://github.com/jomjol/AI-on-the-edge-device/blob/rolling/sd-card/config/config.ini

You could copy the content of that file into your config file. Just change `SetupMode` to false to not enter the setup mode again.

**Reply by mzurhorst** [reply]
It really is strange.  Now,  System/Info is actually showing stuff.  However, I do not have anything in the log file at all.
(I have deleted the log_1970-01-01.txt file because I was hoping to enforce logging into the one with the correct timestamp.  Now it is completely empty.

I will now do the config.ini changes manually and see what happens.

**Reply by mzurhorst** [reply]
Hahaha, I have now copied the first section [TakeImage] into my config.ini, and things are working immediately.
Very strange, esp. since the configuration page is now empty, I only have the "expert mode" checkbox which allows me to edit the config.ini manually.

Anyways, it is now recognizing things.  I will observe this now over the next day and report back.

**Reply by caco3** [reply]
` log_1970-01-01.txt` is used while the device has no NTP time yet.

most parameters have a default value when it is missing in the config file, but maybe not all of them.
